src/Cmfd.cpp

```diff
@@ -1087,57 +1067,64 @@ void Cmfd::setNumFSRs(int num_fsrs) {
 }
 
 
-/** @brief Split the currents of the Mesh cell corners to the nearby surfaces.
- *  @details This method splits the current from each corner evenly to the two
- *           adjoining surfaces.
+/**
+ * @brief Split the currents of the Mesh cell edges to the adjacent faces.
+ * @details This method takes the currents tallied across the edges (or corners)
+ *          of a cmfd cell and splits them evenly across the adjacent faces
+ *          (locations 1 and 2). In order to transport the current through to
+ *          the diagonal cell, the current is also tallied on the surfaces of
+ *          the adjacent cells (locations 3 and 4). Essentially, the tracks that
+ *          cross through edges are split into two half-weight tracks as shown
+ *          in the illustration below:
+ *
+ *                                       |    /
+ *                                       | __/_
+ *                                       |/   /
+ *                                     3 /   /
+ *                                      /|  / 4
+ *                   ------------------/-+-/------------------
+ *                                  1 /  |/
+ *                                   /   / 2
+ *                                  /___/|
+ *                                   /   |
+ *                                  /    |
+ *
  */
-void Cmfd::splitCorners() {
+void Cmfd::splitEdgeCurrents() {
 
-  log_printf(INFO, "Splitting CMFD corner currents...");
+  log_printf(INFO, "Splitting CMFD edge currents...");
 
   int ncg = _num_cmfd_groups;
+  int nf = NUM_FACES;
+  int ne = NUM_EDGES;
+  int ns = NUM_SURFACES;
 
-  #pragma omp parallel
+#pragma omp parallel
```

Comment:
All other preprocessor directives are indented to the line they address. However, the google c++ style guide says that all preprocessor directives should not be indented. This was accidental, but to be consistent with the style guide I would recommend we make a PR that removes the indent from the directives.